torch/csrc/jit/jit_opt_bisect.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+#include <torch/csrc/WindowsTorchApiMacro.h>
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+// `TorchScript` offers a simple bisection utility function
+// that can be configured through environment variable `PYTORCH_OPT_LIMIT`.
```

Comment:
Nit: since it only affect JIT passes, it might makes sense to include `JIT` into the env var name.